raiden/transfer/mediated_transfer/initiator.py

```diff
@@ -151,7 +155,10 @@ def try_new_route(state):
 
 
 def handle_block(state, state_change):
-    state.block_number = state_change.block_number
+    state.block_number = max(
+        state.block_number,
+        state_change.block_number,
+    )
```

Comment:
this means that blocks "older" than the current `state`s block number noop, right? would this here be the place to record a chain reorg?